### PR TITLE
Add Coulomb collision installation to picmi.py

### PR DESCRIPTION
**Pull request by roelof-groenewald**
Support was added to `picmi.py` to allow Coulomb collisions to be installed in simulations executed via Python.
A PICMI version of the existing 2d collisions test was also added to cover the new functionality in CI testing.

### CODE REVIEW

File: Examples/Tests/collision/PICMI_inputs_2d.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+max_steps = 150
+
+max_grid_size = 8
+max_level = 0
+nx = max_grid_size
+ny = max_grid_size
+
+xmin = 0
+xmax = 4.154046151855669e2
```
Comment (dpgrote) [inline Examples/Tests/collision/PICMI_inputs_2d.py:23]: Is this number calculated? Can the expression be used here for clarity? If the expression is not so simple, the number is ok, but include a comment about how it is derived.

File: Examples/Tests/collision/PICMI_inputs_2d.py [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+ymin = xmin
+ymax = xmax
+
+plasma_density = 1e21
+elec_rms_velocity = 0.044237441120300*constants.c
+ion_rms_velocity = 0.006256118919701*constants.c
```
Comment (dpgrote) [inline Examples/Tests/collision/PICMI_inputs_2d.py:29]: Same for these two lines. Can the expression be used instead of the number?

**Comment by roelof-groenewald** [comment]
@dpgrote Thanks for the review. The numbers in the input file were selected to match the numbers from `WarpX/Examples/Tests/collision/inputs_2d` so that the same reference results can be used for the new test as for that one. So I don't actually know how those values were calculated. I changed the input for the RMS velocities to be temperature but really this just replaces one seemingly arbitrary number with another.

**Comment by dpgrote** [comment]
Ok, I see. It would be better then to use the numbers as they are in the input file, adding a comment that these numbers come from inputs_2d.

**Comment by roelof-groenewald** [comment]
> Ok, I see. It would be better then to use the numbers as they are in the input file, adding a comment that these numbers come from inputs_2d.

Sounds good. I'll revert the last commit and add the comments.

**Review by dpgrote — APPROVED** [review]
This looks good! Thanks!

**Pull request merged**